controllers/clusteroperator_test.go

```diff
@@ -266,3 +281,46 @@ func TestUpdateCOStatusDegraded(t *testing.T) {
 		}
 	}
 }
+
+func TestUpdateCOStatusAvailable(t *testing.T) {
+	tCases := []struct {
+		name               string
+		expectedConditions []osconfigv1.ClusterOperatorStatusCondition
+		expected           bool
+	}{
+		{
+			name: "Incorrect Condition",
```

Comment:
This test case seems a little artificial to me. In this case, it might be fine just to test if the condition is what it is supposed to be.
I felt the same about my test code for https://github.com/openshift/cluster-baremetal-operator/pull/29. So, I skipped creating multiple test cases.

Reply:
I don't think its artificial. The fact that the correct output is not expected is a valid test. If I change the expected result to true, then the test will fail.